### PR TITLE
Define missing parameters for mainnet

### LINKED ISSUE #1275: Create the ChainConfig of mainnet
Once everything is done and we're ready to prepare for mainnet, this should be done last.

### PR DESCRIPTION
**Pull request by TheQuantumPhysicist**
Closes #1275

### CODE REVIEW

**Comment by TheQuantumPhysicist** [comment]
So this is now running the pre-mainnet. We should merge it as it has a bunch of fixes.

File: common/src/chain/config/builder.rs [hunk 1]
```diff
@@ -236,6 +268,30 @@ impl Builder {
...
+            super::TESTNET_TOKEN_FREEZE_FEE
+        } else {
+            super::MAINNET_TOKEN_FREEZE_FEE
+        };
+
+        let token_change_authority_fee = if chain_type == ChainType::Testnet {
+            super::TESTNET_TOKEN_CHANGE_AUTHORITY_FEE
+        } else {
+            super::MAINNET_TOKEN_CHANGE_AUTHORITY_FEE
+        };
```
Comment (iljakuklic) [inline common/src/chain/config/builder.rs:293]: Minor: May be a bit nicer to collect these into a sort of table instead of a cascade of if-else.

**Review by iljakuklic — APPROVED** [review]

File: common/src/chain/config/builder.rs (hunk 1 above)
Comment (azarovh) [reply common/src/chain/config/builder.rs:293]: I reworked this in #1472

File: common/src/chain/genesis.rs [hunk 2]
```diff
@@ -34,14 +34,14 @@ pub struct Genesis {
...
     pub fn new(fun_message: String, timestamp: BlockTimestamp, utxos: Vec<TxOutput>) -> Self {
         Self {
-            fun_message,
+            genesis_message: fun_message,
             timestamp,
             utxos,
         }
     }
 
     pub fn fun_message(&self) -> &str {
```
Comment (azarovh) [inline common/src/chain/genesis.rs:43]: `genesis_message()` ?

**Review by azarovh — APPROVED** [review]

File: common/src/chain/genesis.rs (hunk 2 above)
Comment (TheQuantumPhysicist) [reply common/src/chain/genesis.rs:43]: Thanks! Fixing.

**Pull request merged**